spylon_kernel/scala_magic.py

```diff
@@ -29,7 +29,7 @@ class ScalaMagic(Magic):
     def __init__(self, kernel):
         super(ScalaMagic, self).__init__(kernel)
         self.retval = None
-        self._interp = None
+        self._interp = scala_intp
```

Comment:
No, it's not necessary. It's really just to avoid the messaging, etc. around using the magic for the first time if an interpreter already exists.